spring-kafka-test/src/main/java/org/springframework/kafka/test/EmbeddedKafkaKraftBroker.java

```diff
@@ -87,6 +92,24 @@ public class EmbeddedKafkaKraftBroker implements EmbeddedKafkaBroker {
 
 	public static final int DEFAULT_ADMIN_TIMEOUT = 10;
 
+	private static final boolean IS_KAFKA_39_OR_LATER;
+
+	private static final Method SET_CONFIG_METHOD;
+
+	static {
+		IS_KAFKA_39_OR_LATER = ClassUtils.isPresent(CLASS_EXISTS_ONLY_IN_390, EmbeddedKafkaKraftBroker.class.getClassLoader());
```

Comment:
`isPresent()` doesn't throw an exception: can be used directly in the property declaration line.
Also: I don't see a reason in the `CLASS_EXISTS_ONLY_IN_390` constant. You simply can use that class name in this method call.